app/decorators/submission_trait_scores_upload_decorator.rb

```diff
@@ -12,14 +12,47 @@ def parser_summary
           group_by{ |id, scores| scores.size }.
           sort{ |score1, score2| score2.first <=> score1.first }
 
-        summary << " - parsed #{trait_scores.size} plant(s) with unique identification"
+        summary << " - parsed #{trait_scores.size} plant scoring unit(s) with unique identification"
         histogram.each do |scoring_number, plants|
-          summary << "  - #{plants.size} plant(s) have #{scoring_number} trait score(s) recorded"
+          summary << "  - #{plants.size} unit(s) have #{scoring_number} trait score(s) recorded"
         end
       end
 
+      accessions = object.submission.content.step04.accessions
```

Comment:
Would I be able to render it to string in the as_json method of the decorator? I think rendering to string is possible only in controllers, right?